SFBaseKitDemo/SFBaseKitDemo/ExampleLoginModule/LoginCoordinator.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  LogingCoordinator.swift
+//  SFBaseKitDemo
+//
+//  Created by Vesela Ilchevska on 15.04.20.
+//  Copyright © 2020 Upnetix. All rights reserved.
+//
+
+import Foundation
+import SFBaseKit
+
+protocol LoginSceneDelegate: class {
```

Comment:
This `class` might also be `Coordinator`

Reply:
Changed :)